secure_message_v2/views/info.py

```diff
@@ -0,0 +1,26 @@
+import json
+import logging
+from pathlib import Path
+
+from flask import Blueprint, current_app, jsonify, make_response
+from structlog import wrap_logger
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+info_bp = Blueprint("info_bp", __name__, static_folder="static", template_folder="templates")
+
+_health_check = {}
+if Path("git_info").exists():
```

Comment:
Remove this.  It's basically never been useful (as we deploy tagged images) so it'd be nice to discard it instead of dragging it around with us.